src/test/java/com/basho/riak/client/core/util/DefaultCharsetTest.java

```diff
@@ -0,0 +1,49 @@
+package com.basho.riak.client.core.util;
+
+import java.nio.charset.Charset;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+public class DefaultCharsetTest
+{
```

Comment:
As for Runtime, you are totally right, but what about the case when wrong charset specified via Constants.CLIENT_OPTION_CHARSET